app.py

```
@@ -3,7 +3,7 @@
 import pickle
 
 DB_USER = "admin"
-DB_PASS = "SuperSecret123456!"
+DB_PASS = "SuperSecret1234567!"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "DB_PASS"

(S105)